sleap_io/io/dataset.py

```diff
@@ -0,0 +1,279 @@
+from attr import asdict
+import h5py
+import numpy as np
+import json
+from typing import List, Optional
+from sleap_io import (
+    Video,
+    Skeleton,
+    Edge,
+    Node,
+    Instance,
+    LabeledFrame,
+    Track,
+    Point,
+    PredictedInstance,
+)
+
+
+def instance_from_numpy(
+    points: np.ndarray, skeleton: Skeleton, track: Optional[Track] = None
+) -> Instance:
+    """Create an `Instance` from an array of points.
+
+    Args:
+        points: A numpy array of shape `(n_nodes, 2)` and dtype `float32` that
+            contains the points in (x, y) coordinates of each `Node`. Missing nodes
+            should be represented as `NaN`.
+        skeleton: A `Skeleton` instance with `n_nodes` nodes to associate with
```

Comment:
instance -> object (just because instance means something in the SLEAP vocabulary)